sourcePane.js

```diff
@@ -228,7 +228,12 @@ module.exports = {
         .then(function (response) {
           if (!happy(response, 'GET')) return
           const desc = response.responseText
-          textArea.rows = desc ? desc.split('\n').length + 2 : 2
+          if (!desc) { // Defensive https://github.com/linkeddata/rdflib.js/issues/506
+            const msg = 'source pane: No text in response object!!'
+            statusRow.appendChild(UI.widgets.errorMessageBlock(dom, ç))
```

Comment:
@timbl why is this named `ç` shouldn't it be `msg`?

Reply:
@angelo-v -- Probably best to submit a tiny PR for that. Notifications don't always fire properly upon comments/suggestions on already merged PRs.

Reply:
resolved in PR #20